gulpfile.js

```diff
@@ -14,21 +21,96 @@ const styles = () => {
     .pipe(sourcemap.init())
     .pipe(sass())
     .pipe(postcss([
-      autoprefixer()
+      autoprefixer,
+      csso()
     ]))
+    .pipe(rename("style.min.css"))
     .pipe(sourcemap.write("."))
-    .pipe(gulp.dest("source/css"))
+    .pipe(gulp.dest("build/css"))
     .pipe(sync.stream());
 }
 
 exports.styles = styles;
 
+// HTML
+
+const html = () => {
+  return gulp.src("source/*.html")
+  .pipe(htmlmin({ collapseWhitespace: true }))
+  .pipe(gulp.dest('build'));
+}
+
+exports.html = html;
+
+// Scripts
+
+const scripts = () => {
+  return gulp.src("source/js/script.js")
+  .pipe(terser())
+  .pipe(rename("script.min.js"))
+  .pipe(gulp.dest("build/js"))
+}
+
+exports.scripts = scripts;
+
+// Images
+
+const optimizeimages = () => {
+  return gulp.src("source/img/**/*.{svg,png,jpg}")
+  .pipe(gulp.dest("build/img"));
+}
+
+exports.optimizeimages = optimizeimages;
+
+// copyImages
+
+const copyImages = () => {
+  return gulp.src("source/img/**/*.{svg,png,jpg}")
+  .pipe(squoosh())
+  .pipe(gulp.dest("build/img"));
+}
+
+exports.copyImages = copyImages;
+
+// WebImages
+
+const WebImages = () => {
```

Comment:
в JS есть всякие договоренности (об которых ты надеюсь скоро узнаешь). Просто так с большой буквы ничего лучше не писать. На сейчас просто давай напишем это с маленькой буквы 🙂

```suggestion
const webImages = () => {
```